samples/drivers/eeprom/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  name: EEPROM Sample
+common:
+    tags: eeprom
+    integration_platforms:
+      - native_posix
+    harness: console
+    harness_config:
+      type: one_line
+      regex:
+        - "Reset the MCU to see the inceasing boot counter."
+
+tests:
+  sample.drivers.eeprom:
+    tags: eeprom
+    platform_allow: native_posix
```

Comment:
Adding a variant with shield would be nice. Not blocking

Reply:
Added a configuration, but I am not sure about the integration platforms.
I think most boards are already filtered due to console harness?

Reply:
Only native_posix is now listed as integration platform.